### ISSUE TITLE
Frick Collection NYC

### ISSUE THREAD

**Issue by MrChrisWin** [lovasoa/dezoomify#374]
### https://collections.frick.org/objects/275/officer-and-laughing-girl
tried https://collections.frick.org/internal/media/zoomifydispatcher/7337;jsessionid=5D732C901C6290D5E872C1B55ECC1C2E/TileGroup1/5-{{X}}-{{Y}}.jpg

### Uncaught Error: Unable to load tile.
Check that your internet connection is working and that you can access this url:
https://collections.frick.org/internal/media/zoomifydispatcher/7337;jsessionid=5D732C901C6290D5E872C1B55ECC1C2E/TileGroup1/5-0-1.jpg

(https://ophir.alwaysdata.net/dezoomify/zoommanager.js:195)

**Comment by lovasoa** [comment]
The URL to use is `https://collections.frick.org/internal/media/zoomifydispatcher/7337;jsessionid=5D732C901C6290D5E872C1B55ECC1C2E/ImageProperties.xml`
. When you see that tiles URLs contain `TileGroup`, you can replace `TileGroup` and everything that follows to get the image metadata URL to pass to dezoomify.

https://dezoomify.ophir.dev/dezoomify/dezoomify.html#https://collections.frick.org/internal/media/zoomifydispatcher/7337;jsessionid=5D732C901C6290D5E872C1B55ECC1C2E/ImageProperties.xml

**Comment by lovasoa** [comment]
![canvas](https://user-images.githubusercontent.com/552629/78533087-a2764780-77e8-11ea-901a-6908bca60a58.jpg)

**Comment by brushingflamboyant** [comment]
This method seems to work with Vermeer but not with everything. For example this one : https://collections.frick.org/objects/161/the-virgin-and-child-with-st-barbara-st-elizabeth-and-ja

I tried to replace "TileGroup" and the rest after it with "ImageProperties.xml" but an error message occured (invalid XML) : 
https://collections.frick.org/internal/media/zoomifydispatcher/15265/TileGroup0/4-5-5.jpg
https://collections.frick.org/internal/media/zoomifydispatcher/15265/ImageProperties.xml

Do we have to replace with something else?

**Comment by lovasoa** [comment]
Indeed, it looks like they do not make the metainformation URL available. You can still use the dezoomify browser extension on the IIIF version of the image (just click the "iiif" button on the image).

**Comment by brushingflamboyant** [comment]
Thank you, it worked but strangely the image is not fully loaded, there is a "small" part of the image in the upper right corner stuck in it.

![Capture d’écran](https://user-images.githubusercontent.com/70067909/91648449-2c5cd080-ea68-11ea-9b38-79424d4216ff.png)

**Comment by lovasoa** [comment]
Bot-assisted triage note: closing this issue because it is currently labeled `solved`.
